components/reddit/reddit.app.js

```diff
@@ -224,18 +224,21 @@ module.exports = {
         typeahead_active: false,
       };
       const redditCommunities = await this._withRetries(() =>
-        this._makeRequest({
+         this._makeRequest({
```

Comment:
This extra space should not be added as it makes the indentation inconsistent

Reply:
This was a typo but for indentation. Thanks for rising.